packages/react-router-architect/package.json

```diff
@@ -50,7 +50,7 @@
     }
   },
   "dependencies": {
-    "@architect/functions": "^5.2.0",
+    "@architect/functions": "^7.0.0",
```

Comment:
@lpsinger Do you still have concerns for v8 here?
If not, I'd like to update to latest

Reply:
Yes. Please don't update to v8.